File: sbe/binance/spot_3_2/generated-python/spot_sbe/AccountPreventedMatchesResponse.py
"""Generated SBE (Simple Binary Encoding) message codec."""

import struct
from typing import List, Optional, Tuple
from io import BytesIO

class AccountPreventedMatchesResponse:
    """SBE message: AccountPreventedMatchesResponse."""

    TEMPLATE_ID = 403
    SCHEMA_ID = 3
    SCHEMA_VERSION = 2
    BLOCK_LENGTH = 0

    def __init__(self):
        self.prevented_matches = []

    def encode(self) -> bytes:
        """Encode the message to bytes."""
        buffer = BytesIO()


        return buffer.getvalue()

    def decode(self, data: bytes) -> None:
        """Decode the message from bytes."""
        buffer = BytesIO(data)

File: sbe/binance/spot_3_2/generated-php/spot_sbe/AccountPreventedMatchesResponse.php
<?php
/**
 * Generated SBE (Simple Binary Encoding) message codec.
 */

class AccountPreventedMatchesResponse
{
    public const TEMPLATE_ID = 403;
    public const SCHEMA_ID = 3;
    public const SCHEMA_VERSION = 2;
    public const BLOCK_LENGTH = 0;

    public array $preventedMatches = [];

    public function encode(): string
    {
        $buffer = '';


        return $buffer;
    }

    public function decode(string $data): void
    {
        $offset = 0;

    }
}
